app/components/channel_drawer/channel_list.js

```diff
@@ -140,6 +142,10 @@ export default class ChannelList extends React.Component {
         this.props.closeChannelDrawer();
     };
 
+    handleClose = (channel) => {
```

Comment:
Not sure if this has been talked about or not, but is there going to be any kind of protection here against accidental deletion? Something like an alert that confirms that you want to do this?